helix-rest/src/main/java/org/apache/helix/rest/server/resources/AbstractResource.java

```diff
@@ -130,6 +130,13 @@ protected Response OK() {
     return Response.ok().build();
   }
 
+  protected Response OKWithHeader(Object entity, String headerName, Object headerValue) {
+    if (headerName == null || headerName.length()==0)
+      return OK(entity);
+    else
```

Comment:
NIT: remove the else and add {} to if statement. Let's respect our coding convention.